timlubanga/ireport_web
# [Finishes #162109599]add user profile page

What does this PR do?
It enables a user with an account to login to the application
Description of Task to be completed:
view the status individual redflag raised
view the status intervention record


How should this be manually tested
Clone ireport_web into your computer, open the ui folder and run user.html file using any browser. The login page should display

**Any background context you want to provide?
This is just a mockup page. Full functionality is yet to be implemented

What are the relevant pivotal tracker stories?
#162109599,#162109892
